mobile-app/cypress/integration/functional/wallet/loans/loans.spec.ts

```diff
@@ -45,111 +44,6 @@ context.skip('Wallet - Loans', () => {
   })
 })
 
-context('Wallet - Loans Feature Gated', () => {
-  it('should not have loans tab if loan feature is blocked', function () {
```

Comment:
We should replace this with Auctions if we don't have a test now on beta flags.

Reply:
Good Idea, will update it with auction